joplin/api/schema.py

```diff
@@ -723,13 +724,13 @@ def get_structure_for_content_type(content_type):
     if not content_type_data:
         raise Exception(f'content_type [{content_type}] is not included in content_type_map')
 
-    pages = content_type_data["model"].objects.all()
+    pages = content_type_data["model"].objects.filter(live=True)
     for page in pages:
         page_global_id = graphene.Node.to_global_id(content_type_data["node"], page.id)
 
         # Only publish event pages at the date based url
         if content_type == 'event page':
-            if page.date:
+            if page.date and page.live:
```

Comment:
Is this redundant if we're already filtering by live=True?

Reply:
it is. I fixed it, thank you for keeping me honest